src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSOffenderAlias.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.probation.courtcasematcher.restclient.model.courtcaseservice;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.justice.probation.courtcasematcher.restclient.model.offendersearch.OffenderAlias;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CCSOffenderAlias {
+
+    private final LocalDate dateOfBirth;
+    private final String firstName;
+    private final List<String> middleNames;
+    private final String surname;
+    private final String gender;
```

Comment:
if possible, could this be an enum?

Reply:
John and myself had this discussion and made a decision to leave this as string. We do not own this data and is merely used as a cached copy and saved as is as JSON in the DB